rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date
+  attr_reader :person, :book
```

Comment:
- [ ] Kindly ensure that the `Getters` and `Setters` for @book and @person instance variables are present. You can achieve this by adding them to `attr_accessor`. 

```suggestion
  attr_accessor :date, :person, :book
```

Reply:
Thanks. I will change this.